doc/user/data/sql_funcs.yml

```diff
@@ -508,8 +508,8 @@
 
 - type: UUID
   functions:
-  - signature: mz_environment_id() -> uuid
-    description: The `uuid` uniquely identifying this Materialize environment.
+  - signature: mz_environment_id() -> text
+    description: A string containing a `uuid` uniquely identifying this Materialize environment.
```

Comment:
This reads to me as "you parse out the UUID in order to uniquely identify the environment", but in fact it is the whole string that uniquely identifies the environment. (The UUID is the organization's ID, and a given org can have multiple environments.) Maybe just "Returns a string that uniquely identifies this environment"?